__tests__/verify-config/invalid-config--lint-config-off/snapshot.js

```diff
@@ -0,0 +1,6 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`E2E verify-config test with option: { dirName: 'invalid-config--lint-config-off', option: 'off' } 1`] = `
+
+
```

Comment:
Does this option even makes sense? Maybe we can restrict the command to not accept the `off` severity?

Reply:
`off` removed